tap_copper/streams/abstracts.py

```diff
@@ -162,30 +174,42 @@ def get_bookmark(self, state: dict, stream: str, key: Any = None) -> int:
             self.client.config["start_date"],
         )
 
-    def write_bookmark(self, state: dict, stream: str, key: Any = None, value: Any = None) -> Dict:
-        """A wrapper for singer.get_bookmark to deal with compatibility for
-        bookmark values or start values."""
+    def write_bookmark(
+        self,
+        state: dict,
+        stream: str,
+        key: Any = None,
+        value: Any = None,
+    ) -> Dict:
+        """Write bookmark using max(current, value)."""
         if not (key or self.replication_keys):
             return state
 
-        current_bookmark = get_bookmark(state, stream, key or self.replication_keys[0], self.client.config["start_date"])
+        current_bookmark = get_bookmark(
+            state,
+            stream,
+            key or self.replication_keys[0],
+            self.client.config["start_date"],
+        )
         value = max(current_bookmark, value)
         return write_bookmark(
             state, stream, key or self.replication_keys[0], value
         )
 
-
     def sync(
         self,
         state: Dict,
         transformer: Transformer,
         parent_obj: Dict = None,
     ) -> Dict:
         """Implementation for `type: Incremental` stream."""
-        bookmark_date = self.get_bookmark(state, self.tap_stream_id)
+        # Use keyword args to keep pylint error free
+        bookmark_date = self.get_bookmark(
+            state=state, stream=self.tap_stream_id
+        )
```

Comment:
No need to add comment and need proper alignment.

```suggestion
        bookmark_date = self.get_bookmark(
            state=state,
            stream=self.tap_stream_id
        )
```

Reply:
This is Taken care.